SU-HSDO/suhumsci
# Fix: Missing Bourbon Library stylesheet error when compiling

# READY FOR REVIEW

## Summary
- After the percy integration there was a lost patch in `package-lock.json` that allowed the Decanter FE Library to locate and import Bourbon stylesheet. This PR restores this functionality

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. Checkout branch or pull into an existing branch
2. Go to the `humsci_basic` theme directory, delete the existing `node_modules` directory, and run `npm install`
3. Once finished, run `npm run build` and verify there are no sass compiling errors
4. Next run `npm run visreg` and verify Percy is still able to run it's testing

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
